.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Global owner for repo
+* @blink1073 @juliusgeo @ShaneHarvey
```

Comment:
Please add @behackett here since he's the only one experienced in this project.

Reply:
Yes please! :-)

Reply:
Added!